stellar/deploy-contract.js

```diff
@@ -14,26 +14,13 @@ const {
 require('./cli-utils');
 
 async function getInitializeArgs(config, chain, contractName, wallet, options) {
-    const owner = nativeToScVal(Address.fromString(wallet.publicKey()), { type: 'address' });
+    const operator = nativeToScVal(Address.fromString(wallet.publicKey()), { type: 'address' });
 
     switch (contractName) {
         case 'axelar_gateway': {
-            const authAddress = chain.contracts?.axelar_auth_verifier?.address;
-
-            if (!authAddress) {
-                throw new Error('Missing axelar_auth_verifier contract address');
-            }
-
-            return {
-                authAddress: nativeToScVal(authAddress, { type: 'address' }),
-                owner,
-            };
-        }
-
-        case 'axelar_auth_verifier': {
-            const previousSignersRetention = nativeToScVal(15);
             const domainSeparator = nativeToScVal(Buffer.from(arrayify(await getDomainSeparator(config, chain, options))));
             const minimumRotationDelay = nativeToScVal(0);
+            const previousSignersRetention = nativeToScVal(15);
```

Comment:
create an option for this as well, it can default to 15

Reply:
Agreed, updated.